lib/simulator-xcode-6.js

```diff
@@ -178,25 +180,72 @@ class SimulatorXcode6 {
     return _.compact(existence).length !== files.length;
   }
 
+  async getBootedIndicatorString () {
+    let indicator = '';
```

Comment:
This can just be a declaration, since there is no case that will not be assigned in the switch statement.